### ISSUE TITLE
System - Handle/deny ownership change on published collections

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#514]
Currently, ownership change only applies to data and not collections. This will prevent proper changing ownership of data inside published collections since the destination collection will be different. For the time being, deny changing ownership of published data - but must account for the destination collection being published (update moved records pub/cat tags). Eventually, allow for moving of collections (retain contained collections) and permit this on published data.

**Comment by dvstans** [comment]
For now, have just prevented ownership change with published data.

**Comment by JoshuaSBrown** [comment]
If we go with the idea that DataFed is not for publishing data and that can be done with a data catalog won't we avoid having to solve these issues. If we narrow the feature scope of DataFed to private projects etc some of these issues can be removed.

**Comment by JoshuaSBrown** [comment]
For a production quality service, I would prefer to limit the scope as much as possible and further explore this issue possibly in a development version.

**Comment by dvstans** [comment]
Yes, if we remove the catalog feature, this issue is irrelevant.
